lib/jsdom/level2/html.js

```diff
@@ -1270,6 +1270,14 @@ define('HTMLLegendElement', {
   proto: {
     get form() {
       return closest(this, 'FORM');
+    },
+    click: function() {
```

Comment:
That will mean the below test will need to become something like `this.localName === "input" && this.type === "submit"`